app/models/user.rb

```diff
@@ -120,7 +121,7 @@ def search_data
   def invitable_roles
     case role
     when "super_admin"
-      %w[reviewer review_manager super_admin regional_review_manager]
+      %w[reviewer review_manager super_admin regional_review_manager technical_support_local_jurisdiction]
     when "review_manager", "regional_review_manager"
       %w[reviewer review_manager]
```

Comment:
if you want review managers to be able to invite these kinds of users, add it here

Reply:
This has been updated in latest commit